sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/ApproximateUnique.java

```diff
@@ -54,7 +57,9 @@
  *     input.apply(HllCount.Init.forStrings().globally()).apply(HllCount.Extract.globally());
  * }</pre>
  *
- * For more details about using {@code HllCount} and the {@code zetasketch} extension module, see
+ * <p>{@link #combineFn} can also be used manually, with the {@link Combine} transform.
```

Comment:
Please add this new line of comment to the end of the block, because the next line is tied to the previous example.